core/src/main/resources/mysql/mysql_init.sql

```diff
@@ -15,4 +15,20 @@ CREATE TABLE IF NOT EXISTS `metalake_meta` (
     `deleted_at` BIGINT(20) UNSIGNED NOT NULL DEFAULT 0 NULL COMMENT 'metalake deleted at',
     PRIMARY KEY (`metalake_id`),
     UNIQUE KEY `uk_mn_del` (`metalake_name`, `deleted_at`)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin COMMENT 'metalake metadata';
\ No newline at end of file
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin COMMENT 'metalake metadata';
+
+CREATE TABLE IF NOT EXISTS `catalog_meta` (
```

Comment:
What's the name of the database?

Reply:
Currently, it is not mandatory to specify the database name because users are allowed to specify the `jdbcUrl` themselves. Do you think it is necessary to force the database name to be specified?

Reply:
I see, but the SQL files cannot be executed automatically if we skip the scheme here.

Reply:
Yes, my idea is that we do not support automated execution first. We can add an `Entity Store` user doc and clearly describe the initialization steps in it.

Reply:
OK